### PR TITLE
[pytorch] [sagemaker] [test] drop py3 manual check in favor of fixture

### PR DESCRIPTION
**Pull request by ChaiBapchya**
*Issue #, if available:*
Use existing fixture instead of manual check in PT SM test

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON Checklist
* When creating a PR:
- [ ] I've modified `src/config/build_config.py` in my PR branch by setting `ENABLE_EI_MODE = True` or `ENABLE_NEURON_MODE = True`
* When PR is reviewed and ready to be merged:
- [ ] I've reverted the code change on the config file mentioned above
#### Benchmark Checklist
* When creating a PR:
- [ ] I've modified `src/config/test_config.py` in my PR branch by setting `ENABLE_BENCHMARK_DEV_MODE = True`
* When PR is reviewed and ready to be merged:
- [ ] I've reverted the code change on the config file mentioned above

## Reviewer Checklist
* For reviewer, before merging, please cross-check:
- [ ] I've verified the code change on the config file mentioned above has already been reverted

*Description:*

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/sagemaker_tests/pytorch/training/integration/local/test_single_machine_training.py [hunk 1]
```diff
@@ -37,10 +37,8 @@ def test_mnist(docker_image, processor, instance_type, sagemaker_local_session,
 
 @pytest.mark.integration("fastai")
 @pytest.mark.model("mnist")
+@pytest.mark.skip_py2_containers
```
Comment (saimidu) [inline test/sagemaker_tests/pytorch/training/integration/local/test_single_machine_training.py:40]: I think you will have to use

https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/pytorch/inference/conftest.py#L198-L201

Comment (ChaiBapchya) [reply test/sagemaker_tests/pytorch/training/integration/local/test_single_machine_training.py:40]: That's for inference right?
I'm doing for PT SM Training integration local Test 
https://github.com/aws/deep-learning-containers/blob/6c743b1a36d2ab48d13a4472b3fd9e6e4c228aa3/test/sagemaker_tests/pytorch/training/conftest.py#L245-L249

Comment (saimidu) [reply test/sagemaker_tests/pytorch/training/integration/local/test_single_machine_training.py:40]: Ah, sure. My bad. Will resolve this.

**Comment by ChaiBapchya** [comment]
@saimidu 
```
 CapacityError: Unable to provision requested ML compute capacity. Please retry using a different ML instance type.
```
While running
```
integration/sagemaker/test_training_smdebug.py F   
```
Can we retrigger?

**Comment by ChaiBapchya** [comment]
@saimidu Passed the test. good to merge?

**Review by arjkesh — APPROVED** [review]

**Review by saimidu — APPROVED** [review]

**Pull request merged**